pkg/build/linux_linux.go

```diff
@@ -65,6 +68,24 @@ func embedLinuxKernel(params Params, kernelPath string) error {
 	return osutil.CopyFile(imageFile, filepath.Join(params.OutputDir, "image"))
 }
 
+func mountRootPartition(loopFile, mountDir string) error {
+	etcFolder := filepath.Join(mountDir, "etc")
+	if osutil.IsExist(etcFolder) {
+		return nil
+	}
+	for idx := 2; ; idx++ {
+		err := tryMount(fmt.Sprintf("%sp%d", loopFile, idx), mountDir)
```

Comment:
If we mount twice, I think we need to umount twice as well.
It may be better to move copy of sysctl.conf to this function, then we can umount here.

Reply:
Indeed, fixed this.